inspirehep/modules/literaturesuggest/bundles.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014, 2015, 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Bundles for author forms."""
+
+from invenio_assets import NpmBundle
+from invenio_assets.filters import RequireJSFilter
+
+from flask_assets import Bundle
+
+from inspirehep.modules.theme.bundles import js as _js
+
+
+js = NpmBundle(
+    "js/literaturesuggest/inspire_init.js",
+    output="gen/inspire-literature-suggest.%(version)s.js",
+    filters=RequireJSFilter(exclude=[_js]),
+    npm={
+        'buckets-js': '1.98.1',
```

Comment:
Not really, usually better `~1.98.0` or `~1.98.1`, that means that new minor revisions will get installed, just pinning the major revision.